src/main/java/com/solvd/rentals/event/impl/EventServiceImpl.java

```diff
@@ -1,9 +1,9 @@
 package com.solvd.rentals.event.impl;
 
+import com.solvd.rentals.command.ConfirmRentalCommand;
 import com.solvd.rentals.command.CreateRentalCommand;
-import com.solvd.rentals.event.Event;
-import com.solvd.rentals.event.EventService;
-import com.solvd.rentals.event.RentalCreatedEvent;
+import com.solvd.rentals.command.DenyRentalCommand;
+import com.solvd.rentals.event.*;
```

Comment:
@vpetrovaa fix the imports

Reply:
https://github.com/vpetrovaa/rentals/pull/4/commits/59376b65a8fddbf8a6bf275920cd53290739e096